microsoft/testsuites/mshv/mshv_install.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import os
+import re
+from pathlib import Path
+from typing import Any, Dict
+
+from assertpy import assert_that
+
+from lisa import (
+    Environment,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+# from lisa.features import SerialConsole
+from lisa.testsuite import TestResult
+from lisa.tools import Dmesg, Ls, Service, Reboot
+from lisa.util import SkippedException, TcpConnectionException, constants
+from lisa.util.shell import wait_tcp_port_ready
+
+
+@TestSuiteMetadata(
+    area="mshv",
+    category="functional",
+    description="""
+    This test suite is to test VM working well after updating MSHV on VM and rebooting.
+    """,
+)
+class MshvHostInstallSuite(TestSuite):
+
+    _test_hvix_file_path = (
+        Path(os.path.dirname(__file__)) / "test_data" / f"hvix64.exe"
+    )
+
+    _test_hvix_file_path_dst = (
+        Path("/boot/efi/Windows/System32") / f"hvix64.exe"
+    )
+
+    _test_kdstub_file_path = (
+        Path(os.path.dirname(__file__)) / "test_data" / f"kdstub.dll"
+    )
+
+    _test_kdstub_file_path_dst = (
+        Path("/boot/efi/Windows/System32") / f"kdstub.dll"
+    )
+
+    _test_lxhvloader_file_path = (
+        Path(os.path.dirname(__file__)) / "test_data" / f"lxhvloader.dll"
+    )
+
+    _test_lxhvloader_file_path_dst = (
+        Path("/boot/efi") / f"lxhvloader.dll"
+    )
+
+    _init_path_dst = (
+        Path("/home/cloud")
+    )
+
+    @TestCaseMetadata(
+        description="""
+        This test case will
+        1. Update to new MSHV components over old ones in a pre-configured MSHV image
+        2. Reboot VM, check that mshv comes up
+
+        The test expects the MSHV binaries to be installed to be placed under lisa/microsoft/testsuites/mshv/test_data
```

Comment:
Pass in a path as a case variable.

Reply:
Hey @squirrelsc, I'm trying to do just that, referencing this change: https://github.com/microsoft/lisa/commit/44c42cfbc038cc7321a1648e6709f09b580b05de#diff-45aed30957bd1da6be270e89e29ddc7a0acc7af49d6f014aef1fa5d7ddedef6bR82 

Is this the right approach? I'm trying to pass a variable "mshv_binpath" from my runbook to this testcase but it doesn't seem to be working.

Reply:
It's the right way to use the variable. But in the runbook, you need to define a case_visible variable like this:

```yaml
  - name: mshv_bindings_path
    value: ""
    is_case_visible: True
```

Reply:
Worked, thanks!